.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Describe the bug
+A clear and concise description of what the bug is. It should also be clear what the expected outcome is.
+
+## To Reproduce
+Steps to reproduce the bug:
+1. Go to '...'
```

Comment:
Maybe this could use the wrapped [] parts too:?

`1. Go to: [URL or page title]`
`2. Click on [Description of what to click on]`
`3. Scroll down to: [Part of the page with error]`
`4. See error: [Description or screen shot of the error]`

Reply:
I have now wrapped the whole textual part.